### DISCUSSION TITLE
特斯拉機器人 Optimus 將現身？馬斯克預告 AI Day 延至 9 月底

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#649] [General]
**特斯拉執行長馬斯克（Elon Musk）宣布，第二屆特斯拉人工智慧日（AI Day）將延期至 9 月 30 日，屆時 Optimus 人形機器人原型有機會亮相，並設想特斯拉人形機器人能處理乏味無趣的例行工作。**

心得 : 機器人開始取代一些重複繁瑣的工作的未來感覺越來越接近了，雖然一些工作者會被取代，但這機器人也能幫助一些比較高危險性的工作。

全文 : https://technews.tw/2022/06/06/teslas-fake-robot/

**Comment by Yuhsin000** [comment]
高危險且重複性高的工作的確很適合被機器人取代，可以增加效率、減少受傷的機會

**Comment by liangyinglly** [comment]
馬斯克真的是一個能改變世界的狂人 他總是能實現他的想法

**Comment by leon10456** [comment]
感謝樓主的分享 跟者老馬推特投資 別墅靠大海

**Comment by HaoFish123** [comment]
感謝分享，人形機器人我期待好久了。

**Comment by Jimmy911218** [comment]
如果能在往後可以成功的處理顯工作上不只能減少成本也能減少一些作業上的危險

**Comment by ghost** [comment]
雀食 一些工作會被取代是注定的 人類所需要做的是提升自我價值

**Comment by treeann** [comment]
我們只能不斷提升自我競爭力了

**Comment by sys511613** [comment]
無趣的或危險性大的工作就很適合交給機器人來做
現在的人還是去做許多機器人無法辦到的事會更有價值

**Comment by alexthebigpp** [comment]
現在很多重複單一的工作感覺遲早都會被機器人取代，

**Comment by yenwei12138** [comment]
這人型機器人外型也太酷了吧~很好奇他到底能做多少事情

**Comment by bloodtrail0511** [comment]
HI AV灣 艾姆伊隆罵~

**Comment by etin2468** [comment]
他也準備來分機器人這塊餅了阿

**Comment by 110409538** [comment]
人形機器人的確很酷.但如果在晚上突然啟動很容易被嚇到.
